quickstarts/android/03-sessionhandling.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Auth0 Android Quickstarts - 3. Session Handling
+description: This tutorial will show you how to use Lock10 to mantain a session connected.
+name: Android
+alias:
+  - Android
+  - Session Handling
+  - Quickstart
+  - Mobile 
+language:
+  - Android
+hybrid: false
+image: /media/platforms/android.png
+tags:
+  - quickstart
+  - Session Handling
+alias:
+  - android
+seo_alias: android
+---
+
+## Android - Session Handling Tutorial
+
+This tutorial will show you how to use Lock10 to mantain an active session with Auth0.
+
+The user's Credentials, contains three kind of tokens:
+
+* idToken: Identity Token with user information.
```

Comment:
The correct term is identity token that proves the identity of the user. It can be either an opaque token or a jwt. We supply jwt but it's better to add that info as something secondary